policyengine_us/parameters/calibration/gov/hhs/medicare/per_capita_cost.yaml

```diff
@@ -0,0 +1,13 @@
+description: Average Medicare spending per enrolled beneficiary (national average). Represents the annual value of Medicare coverage including Part A and Part B.
+values:
+  2021-01-01: 11_080  # Traditional Medicare (Part A and/or Part B) per KFF
+  2025-01-01: 14_500  # Estimated - update with official CMS data when available
+metadata:
+  label: Medicare per capita cost
+  unit: currency-USD
+  period: year
+  economy: false
+  reference:
+    - title: KFF Medicare Spending Per Beneficiary (2021)
+      href: https://www.kff.org/medicare/state-indicator/per-beneficiary/
+    - title: Note - 2024 value is estimated and should be updated with official CMS data
```

Comment:
We only have a 2021 and 2025 value. We should add one for each year and explain how we did the estimation in 2025?